tests/testing_utils.py

```diff
@@ -41,6 +43,14 @@
 )
 
 
+def repo_name(id: Optional[str] = None, prefix: str = "repo") -> str:
+    """Return a readable pseudo-unique repository name for tests."""
```

Comment:
Could you also provide a code example just to show what it would look like in practice given an input? Something like the following:

```suggestion
    """
    Return a readable pseudo-unique repository name for tests.
    
    Example:
    ```py
    >>> repo_name("my-sace", prefix='space')
    space-my-space-16599481979701
    """
    ```
```